cmd/conformance-tests/scenarios_vsphere.go

```diff
@@ -118,7 +119,7 @@ func (s *vSphereScenario) Cluster(secrets secrets) *apimodels.CreateClusterSpec
 	return spec
 }
 
-func (s *vSphereScenario) NodeDeployments(num int, _ secrets) ([]apimodels.NodeDeployment, error) {
+func (s *vSphereScenario) NodeDeployments(ctx context.Context, num int, _ secrets) ([]apimodels.NodeDeployment, error) {
```

Comment:
Is it needed here?

Reply:
Yes, because some cloud providers need a context, so the interface for scenarios is

```
type testScenario interface {
	Name() string
	Cluster(secrets secrets) *apimodels.CreateClusterSpec
	NodeDeployments(ctx context.Context, num int, secrets secrets) ([]apimodels.NodeDeployment, error)
	OS() apimodels.OperatingSystemSpec
}
```

AWS needs this.

Reply:
Or do you insist on calling the parameter `_` here to signify that it is not used for this scenario? ^^

Reply:
I'd rather name it `_` if it is not used :)

Reply:
Updated.